core/src/main/scala/tech/beshu/ror/configuration/RorIndexNameConfiguration.scala

```diff
@@ -40,7 +40,10 @@ object RorIndexNameConfiguration extends Logging {
   }
 
   def load(esConfig: File): Task[Either[MalformedSettings, RorIndexNameConfiguration]] = Task {
-    new EsConfigFileLoader[RorIndexNameConfiguration]().loadConfigFromFile(esConfig, "Custom ROR index name")
+    new EsConfigFileLoader[RorIndexNameConfiguration]().loadConfigFromFile(
```

Comment:
is it ok? Last time we talked about it I had doubts if this is ok. We use the loadConfigFromFile method, but the description says it's related to "ROR index".

Reply:
This class loads EsConfig JSON from the ES config file and tries to decode the given configuration using provided decoder.
I know the naming used here is misleading, so I refactored it a little bit.